addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -29,7 +29,14 @@ function toggleDropdown($toggles, show) {
                 toShow ? 'shown.bs.dropdown' : 'hidden.bs.dropdown',
                 () => resolve()
             );
```

Comment:
What is this for? shouldn't it also be adapted for the iframe?

Reply:
I was actually added by https://github.com/odoo/odoo/commit/f3e1dc52356164453f72827e3dc87ad903df3d0e
but it's strange because I tested without this code and I can't reproduce the initial bug.

Reply:
This code is needed because the promise has to be resolved.
Caller function do `then()` after the call of this function so yes it seems to be needed.
I adapted it to make it work :+1:

Reply:
The `resolve()` call was never done before. I just follow the [BS documentation about how to register this event ](https://getbootstrap.com/docs/5.3/components/dropdowns/#events)and it works...

Reply:
As discussed, this code is useful for displaying the options of the mega menu when the mega menu is opened. Everything is working correctly in V16 on runbot, so normally we shouldn't need to make any changes with this PR.

Reply:
As discussed, for the second commit, we will just select the dropdown via the iframe to make everything work